yarn-project/aztec-sandbox/docker-compose.yml

```diff
@@ -1,18 +1,18 @@
 version: '3'
 services:
-  fork:
-    image: ghcr.io/foundry-rs/foundry:nightly-a44aa13cfc23491ba32aaedc093e9488c1a6db43
-    entrypoint: 'anvil -p 8545 --host 0.0.0.0 --chain-id 31337'
+  ethereum:
+    image: ghcr.io/foundry-rs/foundry:v1.0.0
+    command: '"anvil --silent -p 8545 --host 0.0.0.0 --chain-id 31337"'
```

Comment:
Fun fact: since Docker is evil and foundry's docker image entrypoint was set to `/bin/sh -c`, all arguments after `anvil` were being passed to `/bin/sh` and not to `anvil` itself. So the whole `-p 8545 --host 0.0.0.0 --chain-id 31337` was being completely ignored and worked by chance just because it happened to match the defaults.

Reply:
Isn't this just an interaction of bash and yaml? :P

Reply:
Fair point, maybe it's not Docker's fault after all!